### PR TITLE
Replace instances of "administrator(s)" with "admin(s)" in docs

### PR DESCRIPTION
**Pull request by aydwi**
## Status

Ready for review

## Description of Changes

As proposed in #2332, this replaces instances of "Administrator(s)/administrator(s)" currently present in docs with "Admin(s)/admin(s)".

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/development/contributor_guidelines.rst [hunk 1]
```diff
@@ -125,7 +125,7 @@ Privileges
 
 .. note:: The privilege escalation workflow is different for
           :ref:`code maintainers <contributor-permissions>` and
-          :ref:`translation maintainers <i18n-administrator-permissions>`.
```
Comment (dachary) [inline docs/development/contributor_guidelines.rst]: Not here.

File: docs/development/getting_started.rst [hunk 2]
```diff
@@ -145,7 +145,7 @@ feel free to reach out to the `translation section of the forum
 <https://forum.securedrop.club/c/translations>`__ or even sub-sections
 where non-English discussions are welcome to facilitate discussions.
 
-As a developer or system administrator you can also help `maintain and
```
Comment (dachary) [inline docs/development/getting_started.rst]: not here

File: docs/development/i18n.rst [hunk 3]
```diff
@@ -297,29 +297,29 @@ We do not want to publish the translator emails so we strip them:
        git log --pretty='%aN' $previous_version..lab/i18n -- \
         securedrop/translations install_files/ansible-base/roles/tails-config/templates | sort -u
 
-.. _i18n-administrator-permissions:
+.. _i18n-admin-permissions:
```
Comment (dachary) [inline docs/development/i18n.rst]: not here

File: docs/development/i18n.rst [hunk 4]
```diff
@@ -297,29 +297,29 @@ We do not want to publish the translator emails so we strip them:
...
 
-Translations administrators
+Translations admins
 ---------------------------
```
Comment (dachary) [inline docs/development/i18n.rst]: make the **-----** line exactly the length of the above string

File: docs/development/i18n.rst [hunk 5]
```diff
@@ -328,10 +328,10 @@ groups. However, the translators community has a different set of
 rules and permissions, reason why it makes sense to have an
 independent policy.
 
-Administrator permissions
+Admin permissions
 ~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (dachary) [inline docs/development/i18n.rst]: make the **~~~~** line exactly the length of the above line

File: docs/development/virtualizing_tails.rst [hunk 6]
```diff
@@ -64,7 +64,7 @@ Next you will install Tails onto the Virtual Hard Disk Image. Start the VM, boot
 to Tails, and enter an administration password and start Tails.
 
 .. note:: For all the instructions that follow, you will need to configure an
-          administrator password each time you boot Tails.
```
Comment (dachary) [inline docs/development/virtualizing_tails.rst]: replace **administrator** with **administration** here because this is how it is called in Tails

**Comment by aydwi** [comment]
@dachary Corrected

**Comment by aydwi** [comment]
@dachary Made the required changes

File: docs/yubikey_setup.rst [hunk 7]
```diff
@@ -21,7 +21,7 @@ Download and Launch the YubiKey Personalization Tool
 ----------------------------------------------------
 
 #. Start Tails. At the log in-screen, choose the option to allow an
-   administrator passphrase.
```
Comment (dachary) [inline docs/yubikey_setup.rst]: s/administrator/administration/ this is what Tails display, we can't change that. Not easily at least.

**Comment by aydwi** [comment]
@dachary Squashed the commits. It automatically requested code owners to review the pull, not sure why :stuck_out_tongue_closed_eyes:

**Comment by dachary** [comment]
@aydwi I'm afraid you squashed more commits than you should have, reason why there are changes from other commits in your squashed commit.

**Comment by aydwi** [comment]
I'll create a new pull

**Pull request closed without merging**